CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This CHANGELOG details important changes made in each version of the
 
 - Terraform-based providers can now communicate detailed information about the difference between a resource's desired and actual state during a Pulumi update.
 - Add the ability to inject CustomTimeouts into the InstanceDiff during a pulumi update.
+- Better error message for missing required fields with default config (#400).
```

Comment:
I *think* this needs to be `[#400](https://github.com/pulumi/pulumi-terraform/issues/400)` to expand correctly?